File: problems/src/test/java/com/palgogo/ex1_ex50/TwoSumTest.java
package com.palgogo.ex1_ex50;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class TwoSumTest {

    @Test
    void twoSum() {
        int[] nums = new int[] {1,3,5,19,4};
        int target = 7;
        TwoSumKt twoSum = new TwoSumKt();
        int[] result = twoSum.twoSum(nums, target);
        assertArrayEquals(new int[]{1,4}, result);

    }
}

File: problems/src/test/kotlin/com/palgogo/ex1_ex50/TwoSumTest.kt
package com.palgogo.ex1_ex50

import org.junit.jupiter.api.Assertions.assertArrayEquals
import org.junit.jupiter.api.Test

class TwoSumKtTestK {

    @Test
    fun twoSum() {
        val nums = intArrayOf(1, 3, 5, 19, 4)
        val target = 7
        val twoSumKt = TwoSumKt()
        val result = twoSumKt.twoSum(nums, target)
        assertArrayEquals(intArrayOf(1, 4), result)
    }
}